### PR TITLE
replace "target system has a 32-bit architecture or a 64-bit architecture" with "version of julia, 32-bit or 64-bit".

### PR DESCRIPTION
**Pull request by nagexiucai**
avoid ambiguity.

### CODE REVIEW

File: doc/src/manual/integers-and-floating-point-numbers.md [hunk 1]
```diff
@@ -56,28 +56,27 @@ julia> 1234
 1234
 ```
 
-The default type for an integer literal depends on whether the target system has a 32-bit architecture
-or a 64-bit architecture:
+The default type for an integer literal depends on version of julia, 32bit or 64bit:
```
Comment (garrison) [inline doc/src/manual/integers-and-floating-point-numbers.md]: hyphens?

Comment (nagexiucai) [reply doc/src/manual/integers-and-floating-point-numbers.md]: Thanks, hyphens had been added.

**Review by nagexiucai — COMMENTED** [review]
these are right now.

**Comment by nagexiucai** [comment]
hi @garrison :)
i am reading through [version 1.0 documents of julia](https://docs.julialang.org/en/v1/) nowadays.
and maybe i will do continuously pull requests about these documents.
if there anything wrong, let me known generously :) duo to i am not so good at github's workflow.

File: doc/src/manual/methods.md [hunk 2]
```diff
@@ -72,17 +72,17 @@ Applying it to any other types of arguments will result in a [`MethodError`](@re
 julia> f(2.0, 3)
 ERROR: MethodError: no method matching f(::Float64, ::Int64)
 Closest candidates are:
-  f(::Float64, !Matched::Float64) at none:1
+  f(::Float64, !Matched::Float64) at REPL[n]:1
```
Comment (fredrikekre) [inline doc/src/manual/methods.md]: Why the change? This will now fail CI since those are tested there.

Comment (nagexiucai) [reply doc/src/manual/methods.md]: oh my omission, thank @fredrikekre :)

Comment (nagexiucai) [reply doc/src/manual/methods.md]: hi @fredrikekre ，i have check the failed [ci](https://travis-ci.org/JuliaLang/julia/jobs/425255392) again, it is a fault caused by REPL[n] indeed.
but i have no idea about why ci did different from illustration above.
functions related are all defined, "Closet candidates" should be "REPL[n]:1" and "Stacktrace" may be "none:0"; if fail into something "Undef~", there only "Stacktrace" which may be "none:0".

**Comment by garrison** [comment]
> and maybe i will do continuously pull requests about these documents.
> if there anything wrong, let me known generously :) duo to i am not so good at github's workflow.

Hi @nagexiucai.  Thanks for the contribution!

It is generally recommended that each pull request make one logical/conceptual change.  So for instance, changing it to say "##-bit julia" everywhere, just as you did originally, is one change that can be debated and merged once it is ready.  But now this pull request seems to include many different unrelated changes, without explanation.  That makes it more difficult to review and will significantly lower the chances it is accepted.

**Comment by garrison** [comment]
The problem is that you are making the pull request from `JulialangOrgCN:master`.  If you look at almost any other pull request to julia, it will be developed on a feature branch, not on `master`.  The idea is that you should have a different feature branch corresponding to each pull request.

For now, you could save your current work onto another branch, and then remove the unrelated commits from this pull request by doing a rebase and a force-push.  And then later you can cherry-pick those commits onto different branches and do pull requests from them.  Or it may be easier for you to close this and start over with a different pull request.

**Comment by nagexiucai** [comment]
okay, thank @garrison so much, a warmhearted pr teaching, i have made a mint :)
now i'd better to close this callow pr and make different branches for each only one logical/conceptual change, and one by one.

File: doc/src/manual/integers-and-floating-point-numbers.md (hunk 1 above)
Comment (nagexiucai) [reply doc/src/manual/integers-and-floating-point-numbers.md]: sorry for troubling @garrison again, bob goes back with #29123 and if this is right way, i will do other pr(s) as well.

File: doc/src/manual/methods.md (hunk 2 above)
Comment (nagexiucai) [reply doc/src/manual/methods.md]: sorry @fredrikekre :( i had seen i went into a incorrect branch as @garrison said.
mybe it is "none:1" in master and i am working at release-v1.0.0 branch.

**Pull request closed without merging**